templates/default/publish.js

```diff
@@ -300,7 +300,13 @@ function buildMemberNav(items, itemHeading, itemsSeen, linktoFn) {
                 itemsNav += '<li>' + linktoFn('', item.name) + '</li>';
             }
             else if ( !hasOwnProp.call(itemsSeen, item.longname) ) {
-                itemsNav += '<li>' + linktoFn(item.longname, item.name.replace(/^module:/, '')) + '</li>';
+                var displayName;
+                if (env.conf.templates.useLongnameInNav)
+                    displayName = item.longname;
+                else
+                    displayName = item.name;
+                itemsNav += '<li>' + linktoFn(item.longname, displayName.replace(/\b(module|event):/g, '')) + '</li>';
```

Comment:
This change looks okay, but a heads up in the pull request would have been appreciated.